release_notes_generator/model/custom_chapters.py

```diff
@@ -41,6 +41,9 @@ def populate(self, records: dict[int, Record]) -> None:
         @return: None
         """
         for nr in records:  # iterate all records
+            if records[nr].skip:
```

Comment:
For better readability I would add a comment here something like "Checks if the record's attr skip is True and skip it if so".

Reply:
Added in commit b4eacda582e8811772da2a9f4633bb3de46c9ce6.